functions/datastore/index.js

```diff
@@ -0,0 +1,135 @@
+// Copyright 2016, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+var gcloud = require('gcloud');
+
+// Create a datastore client.
+var datastore = gcloud.datastore();
+
+// Gets a Datastore key from the kind/key pair in the request
+function _getKeyFromData (data, callback) {
```

Comment:
> Is prefixing with an underscore the way to make things private in GCF?

No, it comes from the JavaScript convention of using underscores to signal that something should be considered private.